### PR TITLE
Add solution

### PR DESCRIPTION
**Pull request by WaveOfDandelions**
GH: https://WaveOfDandelions.github.io/react_movies-list-filter/

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,9 +1,26 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
```
Comment (choeqq) [inline src/App.tsx:8]: I think I would have another state for currently visibleMovies, just to be clear what's is currently displayed

File: src/App.tsx [hunk 2]
```diff
@@ -19,13 +36,17 @@ export const App: React.FC = () => {
                 type="text"
                 id="search-query"
                 className="input"
+                value={query}
                 placeholder="Type search word"
+                onChange={(event) => {
+                  setQuery(() => event.target.value);
+                }}
```
Comment (choeqq) [inline src/App.tsx:43]:
```suggestion
                onChange={handleInputChange}
```

**Review by choeqq — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 2 above)
Comment (WaveOfDandelions) [reply src/App.tsx]: > 

Fixed

File: src/App.tsx (hunk 1 above)
Comment (WaveOfDandelions) [reply src/App.tsx:8]: > I think I would have another state for currently visibleMovies, just to be clear what's is currently displayed

Please provide example if you can

**Review by choeqq — APPROVED** [review]

**Pull request closed without merging**